.changeset/rare-feet-travel.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We can delete this changeset file

Reply:
OK, will delete it.

btw, it was added by the CI bot. Did it detected something wrong?

Reply:
I'm not sure why it added two different changeset files! Will keep an eye out to see if it happens again

Reply:
Sorry, I added this one, and the bot added the another one: `.changeset/hungry-bobcats-own.md`

Should only this file about `@gradio/lite` be deleted, or another one about the `gradio` Python package too?

Reply:
Oh you don't need to add a changeset manually, the bot will take care of it. You can delete the changeset you created.

I don't think we have the automatic changeset support for `@gradio/lite` at the moment. Would be good to add it closer to the release

Reply:
I see.
While it may also work, I have added the changeset for `@gradio/lite` in each PR, like https://github.com/gradio-app/gradio/pull/4779#issuecomment-1620163869, btw.